app/services/provider_authorisation.rb

```diff
@@ -10,17 +10,17 @@ def can_make_offer?(application_choice:, course_option_id: nil)
 
     supplied_course_option = CourseOption.find(course_option_id) if course_option_id
     if supplied_course_option && course_option_id != application_choice.course_option.id
-      application_choice_belongs_to_user_providers?(application_choice: application_choice) && \
+      application_choice_visible_to_user?(application_choice: application_choice) && \
         course_option_belongs_to_user_providers?(course_option: supplied_course_option)
```

Comment:
Should this get a similar treatment? I'm assuming that `supplied_course_option` is there if the offered course is different?

Reply:
We don’t know yet. The offer-a-different course UI is only populated with courses from directly associated providers. In theory an API user could attempt this. Feels like a separate piece of work?